### ISSUE TITLE
MD042 does not detect empty link text

### ISSUE THREAD

**Issue by tv42** [rvben/rumdl#546]
### rumdl version

0.1.58

### Installation method

Other

### Bug description

https://rumdl.dev/md042/ says

> Detects links that are missing either their text or URL,

and includes an incorrect example of

```markdown
[ ](https://example.com)        <!-- Only spaces as text -->
```

but it doesn't actually notice fully empty link text.

(And CommonMark spec shows this syntax as valid: https://spec.commonmark.org/0.31.2/#example-487 )

### Steps to reproduce

```console
printf '# Title\n\n[](#)\n' | rumdl check --config /dev/null --no-cache -
No issues found in <stdin>
```

### Expected behavior

```
<stdin>:3:1: [MD042] Empty link text found: [](#)
```

### Actual behavior

No error.

### Configuration

```toml
# empty
```

### Operating system

Linux

**Comment by rvben** [comment]
Fixed in v0.1.61. Bare `#` URLs (e.g., `[text](#)`) are now correctly treated as empty links by [MD042](https://rumdl.dev/md042). Thanks for reporting, @tv42!
